spec/javascripts/admin/modules/paste_govspeak.spec.js

```diff
@@ -0,0 +1,55 @@
+describe('GOVUKAdmin.Modules.PasteGovspeak', function () {
+  var textarea, pastedGovspeak
+
+  function createHtmlPasteEvent (html = null) {
+    const event = new window.Event('paste')
+    event.clipboardData = {
+      getData: (type) => {
+        if (type === 'text/html') {
+          return html
+        }
+      }
+    }
+
+    return event
+  }
+
+  beforeEach(function () {
+    textarea = document.createElement('textarea')
+
+    pastedGovspeak = new GOVUKAdmin.Modules.PasteGovspeak()
+    pastedGovspeak.start($(textarea))
+  })
+
+  afterEach(function () {
+    textarea.remove()
+  })
+
+  it('should convert simpled formatted content to GovSpeak', function () {
+    textarea.dispatchEvent(createHtmlPasteEvent('<h2>This is a h2</h2>'))
+
+    expect(textarea.value).toEqual('## This is a h2')
+  })
+
+  it('should convert simpled formatted content to GovSpeak', function () {
+    var html = '<h2>This is a h2</h2>'
+    html += '<p>This is a paragraph</p>'
+    html += '<ul>'
+    html += '<li>item 1</li>'
+    html += '<li>item 2</li>'
+    html += '<li>item 3</li>'
+    html += '<li>item 4</li>'
+    html += '</ul>'
+
+    textarea.dispatchEvent(createHtmlPasteEvent(html))
+
+    var expectedText = '## This is a h2\n\n'
+    expectedText += 'This is a paragraph\n\n'
+    expectedText += '- item 1\n'
+    expectedText += '- item 2\n'
+    expectedText += '- item 3\n'
+    expectedText += '- item 4'
+
+    expect(textarea.value).toEqual(expectedText)
+  })
```

Comment:
```suggestion
```

I don't think we should test this here as it is testing the external modules functionality which should be tested by the tool - we're testing it enough with your single test.

Reply:
I was thinking about this and thought I'd add it in anyway. removed